base/applications/calc/whatsnew.txt

```diff
@@ -1,4 +1,30 @@
-1.11 (20080813)
+1.12 (20180806)
+=======================
+* Fixed bug 12.37 13/06/2009 into multiprecision display (unicode only).
+* Added support for average of squares and mean of squares into statistical functions.
+* pop() and push() functions in RPN engines now work with nodes instead of stack units.
+* Updated translations from ReactOS' repository.
+* Added support for new conversion units:
+   - AREA: 'mu', 'rai'.
+   - LENGTH: 'chi', 'cun'. 'sun' optimized.
+   - TIME: 'nanoseconds'.
+   - WEIGHT: 'Kan', 'Kin'.
+* Added support for thailand measure system:
+
```

Comment:
Is there a missing item list there?

Reply:
Ok, since I listed some of the new units before, I will add them too.